#171: Validate duplicate destination table names (#172)
exasol/virtual-schema-common-document: doc/changes/changes_9.4.3.md

# Common Virtual Schema for Document Data 9.4.3, released 2023-??-??

Code name: Validate EDML for duplicate table names

## Summary

> This release validates that the given EDML mapping does not define multiple `destinationTable` entries with the same values because this leads to unexpected behaviour. Documentation is updated accordingly.

Reviewer: ```suggestion
This release validates that the given EDML mapping uses unique values for each `destinationTable` entry as duplicate values led to unexpected behavior. Documentation is updated accordingly.
```
